pkgs/by-name/so/sops/package.nix

```diff
@@ -28,19 +32,34 @@ buildGoModule rec {
     "-X github.com/getsops/sops/v3/version.Version=${version}"
   ];
 
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
 
-  postInstall = ''
-    installShellCompletion --cmd sops --bash ${./bash_autocomplete}
-    installShellCompletion --cmd sops --zsh ${./zsh_autocomplete}
-  '';
+  postInstall =
+    ''
+      installShellCompletion --cmd sops --bash ${./bash_autocomplete}
+      installShellCompletion --cmd sops --zsh ${./zsh_autocomplete}
+    ''
+    + (lib.optionalString (ageWithPlugins != null) ''
+      wrapProgram "$out/bin/sops" \
+          --prefix PATH : "${lib.makeBinPath [ ageWithPlugins ]}"
```

Comment:
Are you sure this is correct? Afaik sops links against the age library and own't pick up this. We rather need the plugins in here. sops need them in PATH instead same way as age.